evergreen/evergreen.yml

```diff
@@ -502,22 +513,24 @@ buildvariants:
   tasks:
     - name: packages-tests
       git_tag_only: true
+      priority: 10
```

Comment:
Set priority for release-related tasks to prioritize package creation over regular tasks.